docs/contributor/02-10-operations.md

```diff
@@ -26,77 +26,77 @@ the prerequisites yourself.
 
-The provisioning process is part of a module reconciliation. To trigger the reconciliation, create a [BtpOperator CR](/api/v1alpha1/btpoperator_types.go):
-
-```shell
-cat <<EOF | kubectl apply -f -
-apiVersion: operator.kyma-project.io/v1alpha1
-kind: BtpOperator
-metadata:
-  name: btpoperator
-  labels:
-    app.kubernetes.io/managed-by: btp-manager
-EOF
-```
-
-The BtpOperator reconciler picks up the created CR and determines whether it should be responsible for representing the
-module status. The BtpOperator CR reflects the status of the operand, that is, SAP BTP Service Operator, only when it is
-the oldest CR present in the cluster. In that case a finalizer is added, the CR is set to `Processing` state and the
+The provisioning process is part of a module reconciliation. 
+1. To trigger the reconciliation, create a [BtpOperator CR](/api/v1alpha1/btpoperator_types.go):
+
+   ```shell
+   cat <<EOF | kubectl apply -f -
+   apiVersion: operator.kyma-project.io/v1alpha1
+   kind: BtpOperator
+   metadata:
+     name: btpoperator
+   EOF
+   ```
+
+2. The BtpOperator reconciler picks up the created CR and determines whether it should be responsible for representing the
+module status. 
+3. The BtpOperator CR reflects the status of the operand, that is, SAP BTP Service Operator, only when it is
+the oldest CR present in the cluster. Otherwise, it is given the `Error` state (3a) with the condition reason `OlderCRExists` and the message containing details about the CR responsible for reconciling the operand.
+4. For the only or the oldest CR present in the cluster,  a finalizer is added, the CR is set to the `Processing` state, and the
 reconciliation proceeds.
-Otherwise, it is given the `Error` state with the condition reason `OlderCRExists` and message containing details
-about the CR responsible for reconciling the operand.
+5. The reconciler looks for a `sap-btp-manager` Secret in the `kyma-system` Namespace with the label `app.kubernetes.io/managed-by: kcp-kyma-environment-broker`. This Secret contains the Service
 
-Next, the reconciler looks for a `sap-btp-manager` Secret in the `kyma-system` Namespace with the label `app.kubernetes.io/managed-by: kcp-kyma-environment-broker`. This Secret contains the Service
 Manager credentials for SAP BTP Service Operator and should be delivered to the cluster by KEB. If the Secret is
-missing, an error is thrown, the reconciler sets `Warning` state (with the condition reason `MissingSecret`) in the CR and stops the reconciliation until the Secret
-is created. When the Secret is present in the cluster, the reconciler verifies whether it contains required data. The
+missing, an error is thrown (5a), and the reconciler sets the `Warning` state (with the condition reason `MissingSecret`) in the CR and stops the reconciliation until the Secret
+is created. 
+6. When the Secret is present in the cluster, the reconciler verifies whether it contains the required data. The
```

Comment:
Starting from point 6 the ordered list formatting is broken. Fixes to point 5 should make it work.